ui/src/pages/Settings.tsx

```diff
@@ -202,17 +208,23 @@ const Form = (): JSX.Element => {
     );
 
     // Update WiFi configuration only if there was a change.
-    if (formState.dirtyFields.wifi) {
+    if (
+      dirtyFields.wifi ||
+      (event?.target.type === 'reset' &&
+        (DEFAULT_VALUES.wifi.accessPointSsid !== wifi?.accessPointSsid ||
+          DEFAULT_VALUES.wifi.accessPointPassword !==
+            wifi?.accessPointPassword))
```

Comment:
what a hassle 🤯

Reply:
When clicking on `Reset`, the WiFi config was not reset.

Reply:
Woow that looks like a pain in the a$$🤣. Alright, let me test it😋.